### DISCUSSION TITLE
convert_bik_to_webm.ps1 - How to convert bik media to webm

### DISCUSSION THREAD

**Discussion by LambdaCore21** [StrataSource/Portal-2-Community-Edition#1064] [Resources]
**convert_bik_to_webm.ps1** is located in `...\steamapps\common\Portal 2 Community Edition\sdk_tools\converter`

1. Extract `ffmpeg-2022-09-22-git-af919cf780-full_build` into something like `C:\Program files...` (using 7-zip)
2. Add the location of `ffmpeg.exe` to your system’s path environment variable (the path should end with `...\ffmpeg-2022-09-22-git-af919cf780-full_build\bin\ffmpeg.exe`) 
![grafik](https://user-images.githubusercontent.com/63379443/197332733-a8241993-3bc9-43ba-ac48-4c9c0e882cd0.png)

3. In PowerShell, use the command cd to navigate to the converter (the path should end with `...\Portal 2 Community Edition\sdk_tools\converter`)
4. In the command prompt, run the command:
`powershell -executionpolicy bypass -File convert_bik_to_webm.ps1 [folder_name]`

Where [folder_name] should be replaced by the relative or absolute filepath of the folder containing the bik’s that you would like to convert to webm’s
(don’t include the square brackets)

**Comment by craftablescience** [comment]
Will update it soon so you don't need ffmpeg on the path, and it can just go next to the script

**Comment by LambdaCore21** [comment]
laura 2
 — 
10/10/2023

it will work with any input video format

`ffmpeg -y -i "input.mp4" -codec:v libvpx-vp9 -crf 12 -b:v 0 -codec:a libvorbis "output.webm"`
